src/vstart.sh

```diff
@@ -968,7 +968,7 @@ $DAEMONOPTS
 $CMONDEBUG
         $(format_conf "${extra_conf}")
         mon cluster log file = $CEPH_OUT_DIR/cluster.mon.\$id.log
-        osd pool default erasure code profile = plugin=jerasure technique=reed_sol_van k=2 m=1 crush-failure-domain=osd
+        osd pool default erasure code profile = plugin=isa technique=reed_sol_van k=2 m=2 crush-failure-domain=osd
```

Comment:
Any reason for changing to m=2? If you are going to increase it you should probably make k and m based on the number of OSDs

Reply:
A previous pull request changed the default in other places to K=2 M=2 so I wanted to make it consistent here. Checking the number of OSDs when making a cluster with vstart is a good idea though.